src/uikit.cs

```diff
@@ -281,29 +332,6 @@ interface NSAttributedString_NSAttributedStringKitAdditions {
 		bool ContainsAttachments (NSRange range);
 	}
 
-	[NoWatch]
-	[iOS (9,0)]
-	[BaseType (typeof(NSObject))]
-	[DisableDefaultCtor] // NSInvalidArgumentException Reason: -[NSDataAsset init]: unrecognized selector sent to instance 0x7f6c8cc0
-	interface NSDataAsset : NSCopying
```

Comment:
Yes, to move it away from a `#if !WATCH`

Reply:
Surrounding this block with an inverted `#if` would have been easier to review I think:

```diff
+#endif // !WATCH
	interface NSDataAsset : NSCopying {
        // ...
        }
+#if !WATCH
```

It results in more conditional code, but I think that's fine in this case (since the amount of lines inside conditional directives decreases).